app/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/testonly/TestOnlyController.scala

```diff
@@ -81,6 +81,7 @@ object TestOnlyController {
       sessionId = sessionId,
       declarationType = DeclarationType.Import,
       maybeExciseOrRestrictedGoods = Some(No),
+      maybeGoodsRouteDestination = Some(No),
```

Comment:
This field are isn't clear - to be fair, the url and page name in the prototype are not very clear either.

The name would need to cover both import and export usage... 

Perhaps something like 'maybeGoodsAreMovingToOrFromTheEUViaNorthernIreland' ?

Reply:
Open to better suggestions for the name!